src/snapshot/snapshot.proto

```diff
@@ -30,7 +30,7 @@ message SnapshotInfo {
   int64 id = 1;
   map <string, string> metadata = 2;
   int64 chunkset_id = 3;
-  string sql_dump_fileset_id = 4;
+  int64 sql_dump_fileset_pin_id = 4;
```

Comment:
It seems like this should be a handle rather than the pin. The pin is, by design, not usable by clients.

Reply:
Removed.